src/data/store/WfsFeatures.js

```diff
@@ -0,0 +1,213 @@
+/* Copyright (c) 2015-2019 The Open Source Geospatial Foundation
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/**
+ * A data store loading features from an OGC WFS.
+ *
+ * @class GeoExt.data.store.WfsFeatures
+ */
+Ext.define('GeoExt.data.store.WfsFeatures', {
+    extend: 'GeoExt.data.store.Features',
+    mixins: [
+        'GeoExt.mixin.SymbolCheck'
+    ],
+
+    /**
+     * The 'service' param value used in the WFS request.
+     * @cfg {String}
+     */
+    service: 'WFS',
+
+    /**
+     * The 'version' param value used in the WFS request.
+     * @cfg {String}
+     */
+    version: '2.0.0',
```

Comment:
I think that `2.0.0` is also the only real version which makes sense, right? As other versions don't support paging as far as I know. Especially since we assume a certain structure in the success-callback. I am a bit unsure whether it was probably doable with previous versions and some vendor specific params.